---
panda_vision/NamedPose[] poses